packages/components/dropdown/src/dropdown.vue

```diff
@@ -137,6 +136,14 @@ export default defineComponent({
       popperRef.value?.onClose()
```

Comment:
We can remove this duplicated code by invoking `handleClose()` here.

Reply:
ok